triton-client/src/components/ProjectCardList/index.tsx

```diff
@@ -1,25 +1,20 @@
+import "../Common.scss"
```

Comment:
Should we have absolute import for that 🤔

Reply:
i dont think we do, i added some additional changes across the board on these types of path